### DISCUSSION TITLE
遇到这个 import local settings failed, ignored 报错该怎么处理？

### DISCUSSION THREAD

**Discussion by phoenixg** [hhyo/Archery#1877] [General]
https://github.com/hhyo/Archery/blob/bc86cda4c3b7d59f759d0d23bb63a54f52616752/archery/settings.py#L335

手工安装的，不是 docker 部署

**Comment by LeoQuote** [comment]
哥，这是个warning，不是个错误

**Comment by phoenixg** [comment]
这个问题似乎是导致 sqladvisor 不能起作用的原因，关联问题：https://github.com/Meituan-Dianping/SQLAdvisor/issues/92

想知道如何解决
